rrap_dg/config.py

```diff
@@ -0,0 +1,47 @@
+from provenaclient import ProvenaClient, Config
+from provenaclient.auth import DeviceFlow
+from pydantic import BaseSettings
+from typing import Optional
+
+
+class Settings(BaseSettings):
+    # The provena deployment to target
+    provena_domain: str = "mds.gbrrestoration.org"
+    # The keycloak realm name
+    provena_realm_name: str = "rrap"
+    # The keycloak client ID
+    provena_client_id: str = "automated-access"
+
+    class Config:
+        env_file = ".env"
+        env_file_encoding = "utf-8"
+
+
+def get_settings() -> Settings:
```

Comment:
Is this function necessary?

Reply:
Not necessary but I like having that there to block the rest of the application from directly depending on a pydantic model which could change interfaces.

Reply:
Ah okay, that makes sense